docs/release-notes/change-log.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Highlights
+
+* Complete redesign of the progress messages: statistics are now aggregated between tier2 and tier1 and represent a snapshot of the current state.
+* tier1 progress messages now include details of current running parallel stages and a breakdown of where the time is spent in each module.
+
```

Comment:
We should mention the deprecation of `ModuleProgressMessage` and telling this don't emit anything anymore.